go/chat/inboxsource.go

```diff
@@ -40,12 +56,14 @@ type RemoteInboxSource struct {
 
 func NewRemoteInboxSource(g *libkb.GlobalContext, boxer *Boxer, ri func() chat1.RemoteInterface,
 	tlf func() keybase1.TlfInterface) *RemoteInboxSource {
+	udc := utils.NewUserDeviceCache(g)
```

Comment:
Is @mlsteele change for a persistent user device cache in master yet?  If so, should this use it?